html/buttons-on-click.html

```diff
@@ -0,0 +1,92 @@
+<!doctype html>
+<html class="no-js" lang="">
+
+<head>
+    <meta charset="utf-8">
+    <title>Redi JS Project</title>
+    <link rel="stylesheet" href="../css/styles.css">
+    <link rel="icon" href="../img/redi.png">
+</head>
+
+<body>
+<div class="header-bar">
+    <ul>
+        <li><a class="active" href="#home">Home</a></li>
+        <li><a href="#about">About us</a></li>
+        <li class="dropdown">
+            <a href="javascript:void(0)" onclick="showAlert(this)">Projects</a>
+            <div class="dropdown-content">
+                <a href="#">Link 1</a>
+                <a href="#">Link 2</a>
+                <a href="#">Link 3</a>
+            </div>
+        </li>
+    </ul>
+</div>
+
+<div class="main-area">
+    <div class="home-content-block">
+        <p>How are you?</p>
+        <button id="awesomeButton">Awesome 🤩</button>
+        <button id="goodButton">Good 👍</button>
+        <button id="badButton">Bad 👎</button>
+        <p id="answer"></p>
+    </div>
+</div>
+
+<div class="footer-bar">
+    <span> Demo project for spring 2020 semester JS class at ReDI Munich </span>
+</div>
+
+<style>
```

Comment:
Why we have inline styles here? why not in css folder? The same for js.

Reply:
The idea was to keep it similar to how it was presented in class, but sure, i can move it to css and js files

Reply:
done